frontend/src/plugin/Plugins.tsx

```diff
@@ -28,13 +28,13 @@ export default function Plugins() {
      *   .initialize() plugins that register (not all do).
      */
     async function fetchAndExecute() {
-      const pluginsScriptPaths = (await fetch(`${helpers.getAppUrl()}plugins/list`).then(resp =>
+      const pluginsScriptPaths = (await fetch(`${helpers.getAppUrl()}plugins/paths`).then(resp =>
```

Comment:
I wonder if some users have been deploying the frontend/backend in a separate way, which would mean that dropping the old route would break the frontend.
(I don't think this is a big issue, but we should not forget to mention it in the release notes)

Reply:
I don't think it's worth mentioning. We've never mentioned other breakages caused by only updating parts of headlamp at once.

If they are using a non standard config, and only upgrade one part of the app... then yeah, it would break.

There's no known instances of anyone deploying things separately anyway. AFAIK.